tsconfig.json

```diff
@@ -1,16 +1,11 @@
 {
   "compilerOptions": {
-    "lib": ["es2017", "dom"],
-    "target": "es6",
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "esModuleInterop": true,
     "noUnusedLocals": true,
-    "rootDir": "src",
     "outDir": "dist",
+    "rootDir": "src",
     "sourceMap": true,
-    "strict": true,
-    "esModuleInterop": true
+    "strict": true
   },
-  "include": ["./src/**/*.ts"],
-  "exclude": ["node_modules", "dist"]
+  "include": ["./src/**/*.ts"]
```

Comment:
This commit could be cleaner if the location of the remaining lines were not altered :)

Reply:
True, I sorted alphabetically the properties to avoid similar problems in the future. This could have been done in a separate commit!